po/fr.po

```diff
@@ -133,7 +132,7 @@ msgstr "Couleurs par défaut"
 #: src/Dialogs/SettingsDialog.vala:131
 msgid "Define the default style used when creating a new shape."
 msgstr ""
-"Définissez le style par défaut utilisé lors de la création d'une nouvelle "
+"Définir le style par défaut utilisé lors de la création d'une nouvelle "
 "forme."
```

Comment:
> Définit

With a -t because it's a description explaining what the settings are for :slightly_smiling_face: